Intersect.Server.Core/Entities/Projectile.cs

```diff
@@ -654,5 +655,10 @@ public override EntityType GetEntityType()
     {
         return EntityType.Projectile;
     }
+    
+    protected override EntityItemSource GetItemSource()
+    {
+        return null;
+    }
```

Comment:
Method isn't nullable, it shouldn't return null. Projectile should just be

```csharp
        return new EntityItemSource
        {
            EntityType = GetEntityType(),
            EntityReference = new WeakReference<IEntity>(this),
            Id = this.Base.Id
        };
```

Same as Resource and Npc right?